hivemind_core/__main__.py

```
@@ -15,18 +15,83 @@
+    from hivemind_core.admin import start_admin_server, init_injected_objects
+    
+    # Try to get database with fallback
+    db = None
+    try:
+        from hivemind_core.database import ClientDatabase
+        from hivemind_core.config import get_server_config
+        
+        cfg = get_server_config()
+        try:
+            db = ClientDatabase()
+        except:
```

Ruff (0.15.5)

[error] Do not use bare `except`

(E722)